app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from typing import List
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: List[Player]) -> float:
```

Comment:
Indicate that return type could be `float` or `int`in the return type annotation. Use `from __future__ import annotations` import in case you face errors
```suggestion
def calculate_team_total_rating(players: List[Player]) -> float | int:
```